src/Validation/Validation.php

```diff
@@ -1424,6 +1424,17 @@ public static function isArray($value)
     }
 
     /**
+     * Check that the input value is a scalar.
+     *
+     * @param mixed $value The value to check
+     * @return bool
+     */
+    public static function isScalar($value)
+    {
+        return is_scalar($value);
```

Comment:
I am find to add tests for nulls. But in my opinion, this method should not pass null, because null is not a scalar, and if I remember correctly, null values are not passed to validation methods and handled specially in Validator class.